plugins/pip/PopupWindow.vala

```diff
@@ -125,7 +125,14 @@ public class Gala.Plugins.PIP.PopupWindow : Clutter.Actor
 		Meta.Rectangle monitor_rect;
 		get_current_monitor_rect (out monitor_rect);
 
-		set_position (SCREEN_MARGIN + monitor_rect.x, monitor_rect.height + monitor_rect.y - SCREEN_MARGIN - height);
+		var x_position = monitor_rect.width + monitor_rect.x - SCREEN_MARGIN - width;
+		var y_position = monitor_rect.height + monitor_rect.y - SCREEN_MARGIN - height;
+
+		if (Clutter.get_default_text_direction () == Clutter.TextDirection.RTL) {
+			x_position = SCREEN_MARGIN + monitor_rect.x;
```

Comment:
Its really a nitpick, so I'm not going to "block" the PR for this, but instead of posibbly setting `x_position` twice, can you do it all inside an if-else statement? :)  You can always declare the variable outside of the if-else block

Reply:
Yeah I agree. I originally wrote what you're suggesting, but went with this cause you can easily remove the RTL logic without effecting it. But, putting the default in an else works. Will do.